### PR TITLE
feat: support minification and source maps

### PR DESCRIPTION
**Pull request by digitalsadhu**
## JIRA Issue
[COREWEB-181](https://jira.finn.no/browse/COREWEB-181)
[COREWEB-82](https://jira.finn.no/browse/COREWEB-82)

## Status
**READY**

## Description

Minification working, can't get source maps working properly due to sourcemap hash changing everytime a bundle occurs which needs to be investigated so leaving sourcemaps for the time being, will ticket up and revisit.

## Todos
- [x] Tests
- [x] Documentation

## Related PRs
* None

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -175,6 +178,8 @@ module.exports = class Router extends EventEmitter {
     postBundleHandler() {
         return async (req, res, next) => {
             const { id, type } = req.params;
+            const sourceMaps = req.query.sourceMaps || false;
+            const minify = req.query.minify || isProduction;
```
Comment (SimenB) [inline lib/main.js]: It would be great to coerce both of these to booleans beforehand

Comment (SimenB) [reply lib/main.js]: also a logical error here - `req.query.minify === 'false'` for instance will be ignored

File: lib/main.js [hunk 2]
```diff
@@ -175,6 +178,8 @@ module.exports = class Router extends EventEmitter {
     postBundleHandler() {
         return async (req, res, next) => {
             const { id, type } = req.params;
+            const sourceMaps = req.query.sourceMaps === 'true';
+            const minify = req.query.minify === 'true' || isProduction;
```
Comment (SimenB) [inline lib/main.js]: This means I cannot pass `'false'`.

Comment (digitalsadhu) [reply lib/main.js]: ah sheeet. Not braining very hard today

**Pull request merged**